configs/openfin/manifest-local.json

```diff
@@ -64,6 +64,10 @@
         "servicesRoot": "http://localhost:3375/finsemble/services",
         "importConfig": [
             "$applicationRoot/configs/application/config.json"
-        ]
+        ],
+        "windowOverrides": {
```

Comment:
@sgd2z can we make it like this? I'm not sure if periods work inside of config items. This layout is much more self-explanatory, and easy to search for:
"monkey-patch":{
   "window.open",
   "window.alert"
}